src/components/navbar/Navbar/ProfileModal.tsx

```diff
@@ -223,14 +232,20 @@ function AccountContent({
 }
 
 function PrivateKeyContent() {
-  const encodedSecretKey = useMyAccount((state) => state.encodedSecretKey)
+  const { encodedSecretKey, getPrivateKey } = useMyAccount()
```

Comment:
same type of issue as my comment with using useMyAccount without attribute selector.

Reply:
Yes, should separate getter to only 1 per call